<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="render_button">
          <property name="text">
           <string>Render</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="up_button">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="down_button">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="add_button">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Active</string>
         </property>
         <property name="icon">
          <iconset theme="address-book-new">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Camera</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Start</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>End</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>X res</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Y res</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Samples</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File \n format</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Engine</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Device</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Motion &lt;br&gt; Blur</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New version</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>High Quality</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Animation Denoise</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Denoise</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Scene</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>View Layer</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Comments</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>View Output</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="icon">
          <iconset theme="call-start">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>396</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actiona_rrp"/>
     <addaction name="actionb_rrp"/>
     <addaction name="separator"/>
     <addaction name="actionDelete_Recent_Files"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDelete_Recent_Files">
   <property name="text">
    <string>Delete Recent Files</string>
   </property>
  </action>
  <action name="actiona_rrp">
   <property name="text">
    <string>a.rrp</string>
   </property>
  </action>
  <action name="actionb_rrp">
   <property name="text">
    <string>b.rrp</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
